packages/desktop-client/src/components/reports/Overview.jsx

```diff
@@ -29,6 +28,20 @@ export function Overview() {
         ...{ paddingLeft: 40, paddingRight: 40, minWidth: 700 },
       }}
     >
+      {customReportsFeatureFlag && (
+        <View
+          style={{
+            flex: '0 0 auto',
+            alignItems: 'flex-end',
+            marginRight: 15,
+            marginTop: 10,
+          }}
+        >
+          <ButtonLink to="/reports/custom" type="primary">
```

Comment:
This is currently navigating to the last loaded custom report rather than to a new custom report if used after looking at a saved report. I think we want it to open a default report each time?

Reply:
I'm not experiencing the same behavior. If I open a saved report then click "reports" in the main menu sidebar then click this button, I get a new default report. Is your experience different? How can I duplicate what you're seeing?

Reply:
Looks like it starts happening if you load a custom report. Then go back to the cards and rename it from there. Then after the rename, click the button. (and it loads with the old name, prior to the rename)

Reply:
I think I got it sorted, can you test again?